release/air_tests/air_benchmarks/mlperf-train/process_imagenet.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Is this file also duplicated with the one in https://github.com/ray-project/ray/pull/37498/files?

Reply:
This one is intentional because we will need it for the torch version of this benchmark that reads directly from images, but it is not used right now in the nightly test.